### DISCUSSION TITLE
For those who Bookmarklets dont work, heres why.

### DISCUSSION THREAD

**Discussion by RavenChill-Dev** [3kh0/ext-remover#781] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
### First, what are Bookmarklets?
Bookmarklets are bookmarks that contain javascript to update a website/webpage, which is a little more easier than executing them in the javascript console with the inspect element. In our case, we want to use bookmarklets to execute certain javascript commands for exploits such as LTBEEF, to open up things like an UI Menu. This is perfectly normal for this case. 
### Ok, then why don't they work?
Ok, so lets say you were trying to perform LTBEEF one day, except, the menu doesn't load, and your like "Hmm, did my school block bookmarklets?" Well, its here's how you can check:

1. First, go to `chrome://policy` to see what your school admins are adding as restrictions.  
2. Then you want to scroll down to "url blocklist" or search it up with ctrl-f, and it should look something like this 
![hah](https://github.com/3kh0/ext-remover/assets/129775544/405e235d-919e-4520-bc72-38c3faeef998)
, and if it says "javascript://" in mono space next to url blocklist, then your school has blocked Bookmarklets :/.  You can check what else your school blocks here as well.
### Ok, now what?
So, currently, there isn't any exploits to re-enable bookmarklets besides unenrollment, but at that point what are exploits for? And using the linux terminal is most likely blocked, so for now, you just have to wait for people like me or mercury workshop to find out how, or try scrolling through bypassi.
If you have any other questions, just ask below, because I was just trying to keep this super simple.

**Comment by funkymokey** [comment]
Microsoft Labs is probably the best and easiest option for people that have strict policy

**Reply by RavenChill-Dev** [reply]
Lol, most likley.

**Reply by ghost** [reply]
yeah or this is new https://github.com/3kh0/ext-remover/discussions/934

**Reply by B3ld000** [reply]
Cool

**Comment by RavenChill-Dev** [comment]
### Also I think this works:
Here is an old exploit that my school blocked, but you guys can try using it.
If your school has Chrome read and right installed, go to:
`chrome-extension://inoeonmfapjbbkmdafoankkfajkcphgd/microsoftpage/mslogin.html`
If it works correctly, you can to "login with github" on it, and then get to google through there, allowing you to have a window with everything unblocked, and you can access now.gg through there for you little kids wanting roblox, and search up https://app.apponfly.com/trial to get to a free windows VM to run exe files like deltarune (Very nice game btw)

**Reply by funkymokey** [reply]
the vms are really good since you can run them on a chromebook (like for example adobe products)

**Reply by DankFireR** [reply]
blocked by chrome RIP

**Reply by DankFireR** [reply]
im just stupid so nvm i realized this required that extension

**Reply by Aka-but-nice** [reply]
mf said read and right :sob::pray: and its patched anyways

**Reply by BasaltDelta** [reply]
bypassi got patched.

**Reply by AshtonDavies** [reply]
> bypassi got patched.

Frr 💀

**Reply by ghost** [reply]
yeah new link is in discussion https://github.com/3kh0/ext-remover/discussions/934

**Reply by ilovetoucans** [reply]
person with google read and write here, link doesnt even work

**Reply by makaikahl2** [reply]
Got patched fr

**Reply by NightmareSR** [reply]
anyway to get past the 20 minute trial for free? Other than resetting it?

**Comment by gooseisgoose** [comment]
if you have microsoft edge you can go to this [link](https://chrome.google.com/webstore/detail/web-code-injector/gffmljieoedhigbaagdjoipecmbiifkf)on edge. and get this extension using the edge extension glitch were edge when your on the chrome web store offers you to get the extension on edge instead. this extension allows you to inject codes into the page might be useful.

**Reply by leahanderson1** [reply]
also works with literally any chromium based browser

**Comment by AshtonDavies** [comment]
It is impossible to work around bookmarklet blocking without external chrome extensions, which are typically also pretty smart at not letting you execute your own code due to webstore policy. Sorry for the utter-most stunning news y'all have ever heard.

edit: rigtools be like...

**Reply by gooseisgoose** [reply]
well the spin page bookmarklet works from the extension whenever I enter or reload a page.

**Reply by RavenChill-Dev** [reply]
I could always just find one of my school admin passwords (Ahem, 100% Not gonna send them a phishing)  and login and change the policy until we get to do whatever we want xd.

**Reply by funkymokey** [reply]
Phishing is already illegal, but phishing your school admin is crazy.

**Comment by RavenChill-Dev** [comment]
I wish I can continue, but im sick :(

**Comment by 3kh0** [comment]
Thanks for the really detailed post for all the skids.

**Reply by Caelan51** [reply]
At this point the creator of LTBEEF needs to release the fucking one that doesn't require the bookmarklets because Skids have no idea on what they'll do because their JavaScript is blocked sadly some people like to be gatekeeping assholes.

**Reply by AshtonDavies** [reply]
@Caelan51 No, there is no reason to continue running the LTBEEF DNS.

**Reply by funkymokey** [reply]
right cus shimboot the goat

**Comment by S-PScripts** [comment]
> 1. chrome

[URLBlocklist](https://chromeenterprise.google/policies/?policy=URLBlocklist)

["chrome://history-frame","chrome://chrome/history-frame","chrome://settings-frame/clearBrowserData","myactivity.google.com/myactivity"]
Cloud
Current user
Mandatory

Javascript not blocked but doesn't work though

**Reply by ghost** [reply]
same issue

**Reply by S-PScripts** [reply]
https://chromium-review.googlesource.com/c/chromium/src/+/4558108
javascript was also banned on managed chromebooks it seems with chromehang and chromekill

**Reply by AshtonDavies** [reply]
> https://chromium-review.googlesource.com/c/chromium/src/+/4558108
> javascript was also banned on managed chromebooks it seems with chromehang and chromekill

Both javascript:// and most debug urls are restricted depending on **devtools availability**.

But it looks like you already linked to that commit which says it all.

**Comment by rwg100** [comment]
stfu we know what bookmarklets are just say "javascript is blocked admin said fuck you"

**Comment by KG210** [comment]
If Linux terminal isn't blocked what do i type to get the bookmarklets to work also it doesn't say java script is blocked

**Reply by S-PScripts** [reply]
https://chromium-review.googlesource.com/c/chromium/src/+/4558108

**Comment by v4pl** [comment]
How is unenrollment not patched?

**Comment by ghost** [comment]
yeash urlblocklist cannot be bypassed

**Comment by ghost** [comment]
https://github.com/3kh0/ext-remover/discussions/934 for new links

**Comment by Charlieis1nsan3** [comment]
their is currently but it is strictly for a specific version of ublock its called ublock origin execution, but as of my chromebook 127 and up its patched T_T
